File: Lista3/Exec1/src/exec1/PessoaJuridica.java
/*  *
 * Classe que cria a Pessoa jurídica a partir da Pessoa
 */
public class PessoaJuridica extends Pessoa{

    double iestadual, cnpj;
    String rsocial;
    /**
     * COnstrutor que a partir da herança da classe pessoa cria a Pessoa Jurídica
     * @param nome
     * @param endereco
     * @param email
     * @param telefone
     * @param cnpj
     * @param insestadual
     * @param razaosocial
     */
    public PessoaJuridica(String nome, String endereco, String email, double telefone, String razaosocial, double insestadual, double cnpj){
        super(nome, endereco, email, telefone);
        this.cnpj = cnpj;
        this.iestadual = insestadual;
        this.rsocial = razaosocial;
    }

    // GETTERS 
    public double getCNPJ(){
        return this.cnpj;
    }
    public double getiestadual(){
        return this.iestadual;
    }
    public double getrsocial(){
        return this.cnpj;
    }

    // SETTER
    public void setCNPJ(double cnpj){
        this.cnpj = cnpj;
    } 
    public void setiestadual(double iestadual){
        this.iestadual = iestadual;
    }
    public void setrsocial(String razaosocial){
        this.rsocial = razaosocial;
    }

}

File: Lista4/exec2/PessoaJuridica.py
from Pessoa import *

class PessoaJuridica(Pessoa):
    def __init__(self, nome: str, cnpj: int, end: str, iscr: str, email: str, rsocial: str):
        super().__init__(nome, end, email)
        self.cnpj = cnpj
        self.iscr = iscr
        self.rsocial = rsocial
        
    def getCNPJ(self):
        return self.cnpj
    def getiscr(self):
        return self.iscr
    def getrsocial(self):
        return self.rsocial
    
    def setCNPJ(self, cnpj):
        self.cnpj = cnpj
    def setiscr(self, iscr):
        self.iscr = iscr
    def setrsocial(self, rsocial):
        self.rsocial = rsocial

